features/events/traps/src/main/java/org/opennms/netmgt/trapd/TrapdConfigBean.java

```diff
@@ -81,5 +85,26 @@ public boolean getNewSuspectOnTrap() {
 	public List<SnmpV3User> getSnmpV3Users() {
 		return m_snmpV3Users;
 	}
+	
+	public void onSnmpV3UsersUpdate(TrapdConfiguration config) {
+		this.m_snmpTrapAddress = config.getSnmpTrapAddress();
+		this.m_snmpTrapPort =config.getSnmpTrapPort();
+		this.m_snmpV3Users=addToSnmpV3User(config.getSnmpv3UserCollection());
+	}
+
+	private List<SnmpV3User> addToSnmpV3User(List<Snmpv3User> snmpv3UserCollection) {
+		List<SnmpV3User> snmpV3UserList=new ArrayList<SnmpV3User>();
+		SnmpV3User snmpV3User = new SnmpV3User();
```

Comment:
The method addToSnmpV3User is written separately since configuration yields us ".org.opennms.netmgt.config.trapd.Snmpv3User" and in Trapconfigbean we have "org.opennms.netmgt.snmp.SnmpV3User" ,Since trapdconfigbean expects SnmpV3User the above method is used to do conversion.